### DISCUSSION TITLE
Where is pytest?

### DISCUSSION THREAD

**Discussion by refervl** [DHI/mikeio#568] [Q&A] [answered]
I want to learn and test mikeio,but I do not find pytest module in tests code.

**Comment by ecomodeller** [comment]
https://docs.pytest.org/en/7.3.x/

**Reply by refervl** [reply]
Thank you for answering!

**Comment by jsmariegaard** [comment, marked as answer]
@refervl if you clone mikeio repo, then install and navigate to the root mikeio folder you should be able to simply write: 

    > pytest

To run all the tests :-) 

Good luck

**Reply by refervl** [reply]
Thank you for answering!
